common/ellipses.js

```diff
@@ -99,11 +101,13 @@
                                                 });
```

Comment:
Do we want to be emitting `error` events at all? (https://github.com/informatics-isi-edu/chaise/blob/rec-errors/common/ellipses.js#L99)